File: server/Models/Recipe.cs
namespace allSpice.Models;

public class Recipe : RepoItem<int>
{

    public string Title { get; set; }
    public string Img { get; set; }
    public string Category { get; set; }
    public string Instructions { get; set; }
    public string CreatorId { get; set; }

    public bool Deleted { get; set; }

    public bool Updated { get; set; }

    public Account Creator { get; set; }

}

File: client/src/models/Recipe.js
export class Recipe {
    constructor(data) {
        this.id = data.id
        this.title = data.title
        this.img = data.img
        this.category = data.category
        this.instructions = data.description

        this.creatorId = data.creatorId //do I need this?
        this.creator = data.creator //or this?

        this.createdAt = new Date(data.createdAt)
        this.updatedAt = new Date(data.updatedAt)
    }
}